testutil-server/src/main/java/io/spine/testing/server/blackbox/verify/state/VerifyState.java

```diff
@@ -18,84 +18,89 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-package io.spine.testing.server.blackbox;
+package io.spine.testing.server.blackbox.verify.state;
 
 import com.google.common.annotations.VisibleForTesting;
-import com.google.common.collect.ImmutableCollection;
 import com.google.common.collect.ImmutableList;
 import com.google.protobuf.Message;
 import io.spine.client.Query;
 import io.spine.client.QueryFactory;
 import io.spine.client.QueryResponse;
 import io.spine.grpc.MemoizingObserver;
+import io.spine.protobuf.AnyPacker;
+import io.spine.server.BoundedContext;
 import io.spine.server.QueryService;
-import org.junit.jupiter.api.Assertions;
 
-import static com.google.common.collect.ImmutableList.copyOf;
+import java.util.Collection;
+
 import static com.google.common.collect.ImmutableList.toImmutableList;
-import static com.google.common.truth.Truth.assertThat;
 import static io.spine.grpc.StreamObservers.memoizingObserver;
-import static io.spine.protobuf.AnyPacker.unpackFunc;
-import static io.spine.testing.client.TestActorRequestFactory.newInstance;
+import static java.util.Collections.singletonList;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
 /**
  * Verifies the states of entities currently present in a bounded context.
  */
 @VisibleForTesting
 public abstract class VerifyState {
 
-    private final Query query;
-    private final ImmutableCollection<? extends Message> expectedResult;
-
-    VerifyState(Query query, ImmutableCollection<? extends Message> result) {
-        this.query = query;
-        expectedResult = result;
-    }
-
     /**
-     * Verifies the entity states.
+     * Queries and verifies entity states.
```

Comment:
I think it's `Queries the read-side of the provided {@code BoundedContext} and {@linkplain here.goes.the.method.reference verifies} the results according to the pre-set expectations`.